.github/workflows/native-image-wasm-spring-shell.yml

```diff
@@ -0,0 +1,43 @@
+name: native-image/wasm-spring-shell
+on:
+  push:
+    paths:
+      - 'native-image/wasm-spring-shell/**'
+      - '.github/workflows/native-image-wasm-spring-shell.yml'
+  pull_request:
+    paths:
+      - 'native-image/wasm-spring-shell/**'
+      - '.github/workflows/native-image-wasm-spring-shell.yml'
+  schedule:
+    - cron: "0 0 1 * *" # run every month
+  workflow_dispatch:
+permissions:
+  contents: read
+jobs:
+  run:
+    name: Run 'native-image/wasm-spring-shell'
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v4
+      - name: Download GraalVM 25.1 EA builds
```

Comment:
Could we use the `graalvm/setup-graalvm` action here?

Reply:
Not yet unfortunately, because we don't have a file for 25.1 in [here](https://github.com/graalvm/oracle-graalvm-ea-builds/tree/main/versions).